src/main/java/org/akip/scheduler/NotifyAssignedTasksScheduler.java

```diff
@@ -0,0 +1,43 @@
+package org.akip.scheduler;
+
+import org.akip.event.JobExecutionTrackingEventControl;
+import org.akip.service.NotifyUserAssignedTasksService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+@ConditionalOnProperty(value = "akip.notify-user-assigned-tasks.enabled", matchIfMissing = true, havingValue = "true")
+public class NotifyAssignedTasksScheduler {
+    private final Logger log = LoggerFactory.getLogger(NotifyAssignedTasksScheduler.class);
+
+    @Autowired
+    private JobExecutionTrackingEventControl jobExecutionTrackingEventControl;
+
+    @Autowired
+    private NotifyUserAssignedTasksService notifyUserAssignedTasksService;
+
+    private static final String CRONNAME = "notifyUserAssignedTasks";
+
+    @Scheduled(cron = ("${akip.notify-user-assigned-tasks.cron-expression}"))
+    public void alertUserAssignedTasks() {
+        String identifier = jobExecutionTrackingEventControl.generateJobExecutionTrackingIdentifier(CRONNAME);
+        Map<String, String> summary = new HashMap<>();
```

Comment:
Move declaration to line 36 (the return of method notify...). In the catch, you can pass as parameter an empty Map